collect.py

```diff
@@ -197,6 +220,10 @@ def run_tests(firefox_driver, chrome_driver):
                not os.path.exists('data/%d_chrome.png' % bug['id']):
                 sequence = run_test(bug, 'firefox', firefox_driver)
                 run_test(bug, 'chrome', chrome_driver, sequence)
+
+                with open("data/" + str(bug['id']) + ".txt", 'w') as f:
+                    f.write(json.dumps(sequence))
```

Comment:
Let's write an object of the sequence per line, instead of a full json object. This way it's easier to see diffs.

Reply:
Ok!